data/search/results.json

```diff
@@ -0,0 +1,315 @@
+                              "name": "icon__uk-parliament",
+                              "data": "header-component.lable"
+                            }
+                        ]
+                      }
+                    }
+                ]
+            }
+        }
+    ],
+    "main-components": [
+        {
+            "name": "section__primary",
+            "data": {
+                "components": [
+                  {
+                      "name": "heading",
+                      "data": {
+                          "heading": ["search.results-heading"],
+                          "size": 1
+                      }
+                  },
+                  {
+                      "name": "form__search",
+                      "data": {
+                        "value": "banana",
+                        "key-word": "search.key-word",
+                        "components": [
+                          {
+                            "name": "icon__search",
+                            "data": "search.search-icon"
+                          }
+                        ]
+                      }
+                  }
+                ]
+            }
+        },
+        {
+            "name": "section__section",
+            "data": {
+                "content-flag": "true",
+                "components": [
+                    {
+                        "name": "heading",
+                        "data": {
+                            "translation": {
+                              "key": "search.about-count",
+                              "data": {
+                                "count": "850"
+                              }
+                            },
+                            "size": 2
+                        }
+                    },
+                    {
+                        "name": "status__highlight",
+                        "data": {
+                            "components": [
+                                {
+                                    "name": "paragraph",
+                                    "data": [
+                                      {
+                                        "text":"status-highlight.new-search"
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
+                    },
+                    {
+                        "name": "list__generic",
+                        "data": {
+                            "type": "ol",
+                            "css-class": "--block",
+                            "components": [
+                                {
+                                    "name": "card__search-result",
+                                    "data": {
+                                        "heading-text": "Result 1",
+                                        "url": "some-url",
+                                        "list-hint": {
```

Comment:
could this be a hint component

Reply:
Also, there is not a limit of just 1 hint - there may be multiple for a given result so `components` containing 1 or more hint may work

perhaps we could even call the attribute `hints`?